client/post-editor/editor-notice/index.jsx

```diff
@@ -190,60 +193,25 @@ export class EditorNotice extends Component {
 				}
 
 				if ( 'page' === type ) {
-					return translate( 'Page scheduled on {{siteLink/}}! {{a}}Add another page{{/a}}', {
-						components: {
-							siteLink: (
-								<a
-									href={ site.URL }
-									target="_blank"
-									rel="noopener noreferrer"
-									onClick={ this.handlePillExternalClick }
-								>
-									{ site.title }
-								</a>
-							),
-							a: <a href={ `/page/${ site.slug }` } />,
-						},
+					return translate( 'Page scheduled for %(formattedPostDate)s.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'Page scheduled for: %s.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).